proc.go

```diff
@@ -99,6 +100,12 @@ func newProc(w, h int) *Proc {
 	proc.cfg.th = material.NewTheme(gofont.Collection())
 	proc.stk.cur().stroke.style.Width = 2
 
+	// If Seed is not called, the generator will produce the same sequence of number every time the
+	// program runs, since the default seed is 1 (see rand.Seed).
+	// Setting a different seed everytime the program starts will make the behavior of Random()
+	// similar to Processing/p5.js.
+	rand.Seed(time.Now().UTC().UnixNano())
```

Comment:
I think we can depart a bit from the p5js behaviour here.

having reproducibility _by default_ goes a long way for debuggability.

also, instead of using the default global `math/rand` that is protected by a mutex, I think we should use a `Proc`-local `Rand` value (as a field of `Proc`).
that field should be initialized with a fixed, known seed (`1` is as good a value as any) and seeded with the `golang.org/x/exp/rand` seed+rand that has better properties.
(the doc comments of `Random` should be modified if we go this route)

WDYT?

Reply:
I agree reproducibility by default will help a lot.  Also after reading this https://github.com/golang/go/issues/21835 I think switching to /x/exp/rand it is a good idea. I will update the PR accordingly.